### PR TITLE
feat: Change to rudder

### PR DESCRIPTION
**Pull request by roneli**

### CODE REVIEW

File: internal/analytics/client.go [hunk 1]
```diff
@@ -122,7 +124,7 @@ func New(opts ...Option) *Client {
 		cfg.Logger = logging.NewSimple(&log.Logger, "analytics")
 	}
 
-	ac, err := analytics.NewWithConfig("NaEpIFd1mc3i6IT7Jas66hp170gNbHzE", cfg)
+	ac, err := analytics.NewWithConfig("28gqMiozjM69W0Tr4wqA1CVPOXa", "https://cloudquerypgm.dataplane.rudderstack.com", cfg)
```
Comment (bbernays) [inline internal/analytics/client.go:127]: Do we want to have a different production key? Insert this via `ldflags` at compile time?

**Review by bbernays — APPROVED** [review]
LGTM other than a single comment

**Review by bbernays — APPROVED** [review]
LGTM. Like the ability to test out new API keys directly from the CLI

**Review by disq — APPROVED** [review]
LGTM

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**